src/textual/geometry.py

```diff
@@ -373,16 +422,16 @@ def intersection(self, region: Region) -> Region:
         rx2 = cx2 if x2 > cx2 else (cx1 if x2 < cx1 else x2)
         ry2 = cy2 if y2 > cy2 else (cy1 if y2 < cy1 else y2)
 
-        return Region(rx1, ry1, rx2 - rx1, ry2 - ry1)
+        return Region(rx1, ry1, (rx2 - rx1), (ry2 - ry1))
```

Comment:
More braces! Do you find these clearer or something?

Did you run black on this?

Reply:
Yeah, I do personally find them more readable, but Black takes them right out again. Sorry for wasting your time.